CHANGELOG.md

```diff
@@ -3,4 +3,4 @@
 ## v1.0.0 (September 2020)
 
 This is the first release of an MQTT client library in this repository.
-It has full compliance with the [MQTT 3.1.1 specification](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc385349213).
\ No newline at end of file
+The MQTT library is a client-side implementation of the [MQTT 3.1.1 specification](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc385349213), suitable for resource-constrained devices.
```

Comment:
Eliminate the `#_Toc385...` at the end of the URL, this directs the user to deep within the page.